main/src/ca/uwaterloo/flix/language/ast/DesugaredAst.scala

```diff
@@ -21,6 +21,8 @@ import ca.uwaterloo.flix.language.ast.Ast.Denotation
 import ca.uwaterloo.flix.language.ast.shared.Fixity
 import ca.uwaterloo.flix.util.collection.MultiMap
 
+import java.lang.reflect.Field
```

Comment:
Unused import?

Reply:
Looking at the diff, this seems to have happened in every AST :)

Reply:
It is not unused because the `fields` field of the `Struct` case has a `Map` of fields whose value type is `Field` as suggested in #7934

Reply:
Java reflection fields? Maybe it should be Name dot Field?

Reply:
`Name.Field` is the key type. I can make a new `StructField` type?

Reply:
This is a map from field names to field values. `Name.Field`(I called it `Name.StructField` but that can be changed) is the key type of the map but not the value type. The value type I think should contain information about the struct field(probably a type and a location). You're right, the java reflection field doesn't make sense. I think I should add a `StructField` case to the AST and use that type.